ios/RNSentry.m

```diff
@@ -266,4 +266,10 @@ + (BOOL)requiresMainQueueSetup {
     [SentrySDK crash];
 }
 
+RCT_EXPORT_METHOD(closeNativeSdk
+                  : (RCTPromiseResolveBlock)resolve rejecter
+                  : (RCTPromiseRejectBlock)reject) {
+  [[SentrySDK currentHub] getClient].options.enabled = NO;
```

Comment:
see this https://github.com/getsentry/sentry-dart/pull/424/files#r617398293
using latest Cocoa beta

Reply:
Made it a separate PR #1459